llvm-spirv/test/transcoding/ForwardPtr.ll

```diff
@@ -3,11 +3,12 @@
 ; RUN: llvm-spirv -r %t.spv -o %t.bc
 ; RUN: llvm-dis %t.bc -o - | FileCheck %s --check-prefix=CHECK-LLVM
 
-; CHECK-SPIRV: TypeForwardPointer [[#FwdPtr:]] [[#SC:]]
-; CHECK-SPIRV: TypeFunction [[#]] [[#]] [[#FwdPtr]]
+; CHECK-SPIRV: TypeForwardPointer [[#FwdPtr:]] 8
```

Comment:
Why did you hardcode `StorageClass` operand here?

Reply:
I'll try to answer the question, once I build the compiler with this patch. If I remember correctly, this LIT started to fail on Windows, but I'm unsure why.

Reply:
Yes, there are 2 x TypeForwardPointer and on windows checks are being confused by this. I could add another empty check for TypeForwardPointer to skip it, but it would make a test less stable in case of order of TypeForwardPointer generation changed.

Reply:
Ok, that let's leave this as-is for now